src/json-patch-duplex.ts

```diff
@@ -471,17 +470,17 @@ module jsonpatch {
       if (obj.hasOwnProperty(key) && !(obj[key] === undefined && oldVal !== undefined && _isArray(obj) === false)) {
         var newVal = obj[key];
         if (typeof oldVal == "object" && oldVal != null && typeof newVal == "object" && newVal != null) {
-          _generate(oldVal, newVal, patches, path + "/" + escapePathComponent(key));
+          _generate(oldVal, newVal, patches, path + "/" + escapePath(key));
```

Comment:
I don't understand why this was renamed, since it behaves like `encodeURIComponent`, not `encodeURI` - you can't pass it a whole pointer, only components of a pointer